solidity/contracts/BancorNetwork.sol

```diff
@@ -56,12 +56,12 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, F
     /**
       * @dev triggered when a conversion between two tokens occurs
       * 
-      * @param _smartToken      smart token governed by the converter
-      * @param _fromToken       ERC20 token converted from
-      * @param _toToken         ERC20 token converted to
-      * @param _fromAmount      amount converted, in fromToken
-      * @param _toAmount        amount returned, minus conversion fee
-      * @param _trader          wallet that initiated the trade
+      * @param _smartToken  smart token governed by the converter
+      * @param _fromToken   source ERC20 token

```

Comment:
@yudilevi : Did you miss these two, or would it break current API?

Reply:
yeah, will break the API